manifests/1.3.17/opensearch-dashboards-1.3.17.yml

```diff
@@ -9,31 +9,31 @@ ci:
 components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
-    ref: '1.3'
+    ref: 4bee12f551c414aff10c58359910b6d20b45570d
   - name: functionalTestDashboards
     repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
-    ref: '1.3'
+    ref: 2c5dc0b2bb5e2f979034085f84d61900ce6dc021
```

Comment:
Do we want to keep this as `1.3`? Since its not bundled and test changes can be picked up?

Reply:
Done!